src/awkward/_v2.py

```diff
@@ -0,0 +1,22 @@
+# BSD 3-Clause License; see https://github.com/scikit-hep/awkward-1.0/blob/main/LICENSE
+
+# Following https://github.com/scikit-hep/awkward/blob/main-v1/src/awkward/_v2/__init__.py
+
+from awkward import Array  # noqa: F401
+from awkward import ArrayBuilder  # noqa: F401
+from awkward import Record  # noqa: F401
+from awkward import _broadcasting  # noqa: F401
+from awkward import _connect  # noqa: F401
+from awkward import _lookup  # noqa: F401
+from awkward import _slicing  # noqa: F401
+from awkward import _typetracer  # noqa: F401
+from awkward import _util  # noqa: F401
+from awkward import behavior  # noqa: F401
+from awkward import behaviors  # noqa: F401
+from awkward import contents  # noqa: F401
+from awkward import forms  # noqa: F401
+from awkward import identifier  # noqa: F401
+from awkward import index  # noqa: F401
```

Comment:
Should we also expose `numba.py`, etc.? Looking at [v2 in main-v1](https://github.com/scikit-hep/awkward/tree/main-v1/src/awkward/_v2), we have a few modules that aren't listed here:
- `numba`
- `operations`
I've determined this just by looking for `import awkward.XXX` - this will set `XXX` on the awkward object.
```suggestion
from awkward import index  # noqa: F401
from awkward import numba  # noqa: F401
from awkward import operations  # noqa: F401
```

Reply:
If you are happy to make this change, then provided it works with uproot, I'm happy to merge!